src/main/java/hudson/plugins/jacoco/model/CoverageObject.java

```diff
@@ -203,7 +204,7 @@ public Coverage getLineCoverage() {
 	/**
 	 * Gets the build object that owns the whole coverage report tree.
 	 */
-	public abstract AbstractBuild<?,?> getBuild();
+	public abstract Run<?,?> getBuild();
```

Comment:
@amuniz points out the withBridgeMethods - but if there are no external users and the maintainer is ok with breaking the API - I would just break the API and reduce the scope wherever possible.